src/components/pages/ManageResources/RenderResourceManagement.js

```diff
@@ -240,11 +234,7 @@ export const RenderResourceManagement = () => {
         </Form.Item>
         <Form.Item
           name="pertains_to"
-          label={
-            <label style={{ color: 'black', padding: '0rem' }}>
-              Pertains to
-            </label>
-          }
+          label={<label style={{ padding: '0rem' }}>Pertains to</label>}
```

Comment:
I like the changes for each of these--much simpler to see on one line instead of five.